.evergreen/buildvariants-and-tasks.in.yml

```diff
@@ -443,6 +455,7 @@ tasks:
       - assigned_to_jira_team_compass_compass
       - foliage_check_task_only
     commands:
+      - func: screencapture
```

Comment:
Adding a call to a new `screencapture` function here to exit early on MacOS if the host isn't logged into a graphical shell.